tests/console/cups.pm

```diff
@@ -35,6 +35,7 @@ sub run {
     services::cups::config_service();
     services::cups::enable_service();
     services::cups::start_service();
```

Comment:
This means that if cups wasn't running previously, it's started and then immediately restarted. That's a waste of time and space and might actually break it if it gets forcibly terminated during initial startup somehow.

So I suggest to just restart instead of start.

Reply:
Right, that might be a problem. Fixed, thanks!